docs/pages/modules/third-party-library.mdx

```diff
@@ -94,6 +95,46 @@ dependencies {
     # Swift/Objective-C compatibility
 ```
 
+<Collapsible summary="Are you trying to use a .framework dependency?">
+
+On iOS you can also use dependencies bundled as a framework by using the `vendored_framework` config option.
+
+```diff ios/ExpoRadialChart.podspec
+    s.static_framework = true
+    s.dependency 'ExpoModulesCore'
++   s.vendored_frameworks 'Frameworks/MyFramework.framework'
+    # Swift/Objective-C compatibility
+```
+
+</Collapsible>
+
+<Collapsible summary="Are you trying to use a .aar dependency?">
+
+Create a `libs` folder in the `android` folder of your Expo module. Place your `aar` file there.
+
+Next add the folder as a repository like so:
+
+```diff android/build.grade
+  repositories {
+    mavenCentral()
++   flatDir {
++       dirs 'libs'
++   }
+  }
+```
+
+Last, add the dependency to the `dependencies` list (using a package path, not a filename, but note the `@aar` at the end):
```

Comment:
```suggestion
Finally, add the dependency to the `dependencies` list. Instead of the filename, use the package path, which includes the `@aar` at the end:
```

Improved verbiage here as to what final step to take, and also the info inside the brackets should be part of the main flow so that a developer doesn't ignore it.